javascript-stringify.js

```diff
@@ -214,7 +214,10 @@
     'string': function (string) {
       return "'" + string.replace(ESCAPABLE, escapeChar) + "'";
     },
-    'number': String,
+    'number': function (val) {
+      if (val === 0 && 1 / val === -Infinity) return '-0';
```

Comment:
What about using `Object.is`? I realize this has much better compatibility though, so LGTM!

Reply:
Indeed, I believed `Object.is` was not part of Node 4, but it is.